src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -27,25 +27,42 @@
 from pybuilder.errors import BuildFailedException
 from pybuilder.utils import assert_can_execute
 from pybuilder.utils import execute_command
+from pybuilder import scaffolding as SCAFFODING
+from pybuilder.core import NAME_ATTRIBUTE
 
-__author__ = 'Thomas Prebble'
+__author__ = 'Thomas Prebble', 'Marcel Wolf'
 
 
 use_plugin("core")
 
 
-DEFAULT_SPHINX_BUILDER = "html"
-DEFAULT_SPHINX_SOURCE_DIR = ""
-DEFAULT_SPHINX_OUTPUT_DIR = "_build/"
+DEFAULT_SPHINX_OUTPUT_DIR = SCAFFODING.DEFAULT_DOCS_DIRECTORY + "/_build/"
+
+"""
+                           build.py:
+    :param ${name}:        name = "project name"
+    :param ${version}:     version = "0.01"
+    :param ${doc_author}   project.doc_author = "project team"
+    :param ${doc_builder}  project.doc_builder = "html"
+
+"""
+
+AUTHORS = '${doc_author}'
```

Comment:
filter_resources will replace these placeholders with the pybuilder metadata when we build a pybuilder release. As a result everyone installing the pybuilder release will have no way to change this metadata.